recipe/build.sh

```diff
@@ -2,7 +2,15 @@
 
 set -ex
 
+# Install WASI SDK
+export WASI_VERSION=22
```

Comment:
I think we have tried with `WASI SDK` version 19 on my system and version 21 on Ondrej's system. So, maybe let's try/use one of these?